src/ranged.cpp

```diff
@@ -1380,6 +1382,20 @@ void make_gun_sound_effect(player &p, bool burst, item *weapon)
     }
 }
 
+void make_gun_flash(player &p , item *weapon)
+{
+    if( p.weapon.is_gun()  && !p.weapon.is_charger_gun()) {
+        if(p.weapon.has_gunmod("suppressor") >= 0 || p.weapon.has_gunmod("crafted_suppressor") >= 0 ||
+           p.weapon.has_gunmod("shot_suppressor") >= 0) {
+            return;
+        }
+        if( rng(0,3) < 1  ) {
```

Comment:
Ha, that's a good one, made me chuckle (-:

Even the cataclysm didn't change the laws of math: `rng(0,3)` return one of 0,1,2,3 - neither of those is *ever* less than 1, this condition is never true.

Btw. for this kind of check, you can also use the `one_in` or `x_in_y` functions, e.g. `one_in(4)` instead of `rng(0,3) > 0`.